### ISSUE TITLE
Add to Deployment Pipeline on CircleCI

### ISSUE THREAD

**Issue by gshaw1997** [Cyber4All/CLARK-Gateway#4]
Add a circle.yml to root of the repository to add instructions for deploying production branch to AWS Elasticbeanstalk.
 
Will need to add command to copy over package.json to dist folder once built.

**Comment by gshaw1997** [comment]
Example circle.yml

```machine:
  node:
    version: 7.9.0
dependencies:
  pre:
    - npm install
    - npm install enhanced-resolve@3.3.0
test:
  override:
    - echo "done"
deployment:
  production:
    branch: production
    commands:
      - ng build --prod --aot=false
      - 'aws s3 sync ${source} s3://${s3_bucket} --region ${s3_region}'
```

**Comment by DonSeannelly** [comment]
Closed with 1bf2bb5
